doc/tools/docgram/orderedGrammar

```diff
@@ -1463,7 +1471,7 @@ simple_tactic: [
 | "evar" "(" ident ":" term ")"
 | "evar" one_term
 | "instantiate" "(" ident ":=" term ")"
-| "instantiate" "(" int ":=" term ")" OPT hloc
+| "instantiate" "(" integer ":=" term ")" OPT hloc
```

Comment:
Idem, it could be `natural` (for an integer, the code says `if n <= 0 then user_err Pp.(str "Incorrect existential variable index.")`.